### ISSUE TITLE
Can you confirm firebase_core is no longer needed?

### ISSUE THREAD

**Issue by stephenhuey** [FirebaseExtended/flutterfire#2470]
It seems I'm able to use cloud_firestore and firebase_auth without firebase_core in Flutter, but this list of dependencies implies it's necessary:

https://pub.dev/packages?q=dependency%3Afirebase_core

I did just find these links which seem to confirm it is no longer needed: 

https://firebase.google.com/support/release-notes/android

https://firebase.google.com/docs/ios/setup

Can someone who knows confirm that firebase_core is not needed to use the databases, auth, analytics, etc?  Just don't want to end up missing something in production when in dev everything seems fine!  :)

**Comment by iapicca** [comment]
Hi @stephenhuey 
the link you provided refer to the native `android` and `ios` plugins
for flutter plugins might not always be the case
Please see https://firebase.google.com/support for resources for asking questions like this
and check our [guides](https://firebaseopensource.com/projects/firebaseextended/flutterfire/);
you may also get some help if you post it on Stack Overflow.
Closing, as this isn't an issue with FlutterFire itself,
if you disagree please write in the comments and I will reopen it
Thank you

**Comment by stephenhuey** [comment]
Funny, @iapicca is the one who asked me to open this question in a dedicated GitHub when I asked this in the Flutter one:

https://github.com/flutter/flutter/issues/53835

If the FlutterFire one is not dedicated enough, which one is?  As I implied, the guides tend to be out-of-date which is why I was raising the issue.  If the answer is to open a support case, I can do that, but this is a rather convoluted roundabout way to correcting misleading or incomplete documentation.  Or maybe there's an unexplained reason to add core even though things appear to work without it...?

**Comment by iapicca** [comment]
Hi @stephenhuey 
surely the issue on flutter/flutter
> The following page says "Note: All Flutter-Firebase apps, both iOS and Android versions, require the firebase_core plugin for Flutter."
> 
> https://firebase.google.com/docs/flutter/setup
> 
> But firebase_core is missing from this page of the Firebase for Flutter codelab:
> 
> https://codelabs.developers.google.com/codelabs/flutter-firebase/#3
> 
> This is all it mentions adding to your project:
> 
> `dependencies: flutter: sdk: flutter cloud_firestore: ^0.13.0+1 # new`
> 
> Shouldn't firebase_core also be there?

needs to be addressed in the current github repo,
nevertheless your present question is rather different;
did you mean the issue as proposal to specify in the documentation
in which cases [firebase_core](https://pub.dev/packages/firebase_core) is required?
thank you

**Comment by stephenhuey** [comment]
Yes, although maybe it's not that important.  I just realized that the other packages seem to be pulling down core even if I don't explicitly put it into pubspec, so perhaps this issue is moot (people will experience the same intended result whether or not firebase_core is put into the pubspec).  Pardon the trouble, but it's probably not worth me raising a ruckus about.  :)

I was just clarifying so that I don't run into trouble in prod, but I think I've got it!
